meta/classes/base.bbclass

```diff
@@ -99,6 +99,10 @@ base_set_conf_value() {
    base_adjust_conf_value "$1" "$2" "$3" "="
 }
 
+base_set_weak_conf_value() {
```

Comment:
IMO, this is weak-weak `??` assignment, not just weak `?`

Reply:
As I see in documentation, "weak" assignment is `??=` and "default" asignment is `?=`.
Please see 
* "3.1.5. Setting a default value (?=)" in https://www.yoctoproject.org/docs/latest/bitbake-user-manual/bitbake-user-manual.html#setting-a-default-value
* "3.1.6. Setting a weak default value (??=)" in https://www.yoctoproject.org/docs/latest/bitbake-user-manual/bitbake-user-manual.html#setting-a-weak-default-value
* Example of usage in Reference manual - "Because the default is set using a "weak" assignment (i.e. "??=")" in https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-ROOT_HOME

Reply:
sounds good